lib/ts/types.ts

```diff
@@ -113,3 +114,13 @@ export type User = {
     // the recipeUserId can be converted to string from the RecipeUserId object type.
     toJson: () => any;
 };
+
+export type SSRSessionContextType = {
+    isContextFromSSR: true;
+    loading: false;
```

Comment:
These should not be required since they are static values.

Reply:
But they are expected by `supertokens-auth-react`, if they will be optional - that means we would have to polyfill them inside `SessionAuth` component, and in that case I would rather remove them completely from `supertokens-node`